qt/launcher-specific

```diff
@@ -10,6 +10,13 @@ export QTCHOOSER_NO_GLOBAL_DIR=1
 if [ "$USE_qt5" = true ]; then
   export QT_SELECT=snappy-qt5
 elif [ "$USE_qt5-app-platform" = true ]; then
+  ls -d $SNAP/ubuntu-app-platform >/dev/null 2>&1
+  if [ $? -ne 0 ] ; then
```

Comment:
It's just a matter of preference, but can you just condense that in one line (just use backticks so that it doesn't depend on bash), as in other part of the launcher, thanks!